Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/User-Picker/index.md

```diff
@@ -0,0 +1,35 @@
+---
+versionFrom: 8.0.0
+---
+
+# User Picker
+
+`Alias: Umbraco.UserPicker`
+
+`Returns: IPublishedContent`
+
+The user picker opens a panel to pick a specific user from the Users section. The value saved is of type IPublishedContent.
+
+## Data Type Definition Example
+
+![Media Picker Data Type Definition](images/User-Picker-DataType-v8.png)
+
+## Content Example
+
+![Member Picker Content](images/User-Picker-Content-v8.png)
+
+## MVC View Example
+
+Please note that getting the Value of the property will simply return a user ID - properties of the User can be accessed by referencing UserService.
```

Comment:
We try not to use words like `simply` in the docs.. I've tried a small rewrite of this sentence. What do you think? 😁 

```suggestion
Please note that getting the Value of the property will return the user ID - properties of the User can be accessed by referencing UserService.
```